docs/deployment/private-ddn/provision-a-data-plane.mdx

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 1
+sidebar_label: Provision a Data Plane
+description: "Learn how to provision a private Data Plane."
+keywords:
+  - hasura ddn
+  - private ddn
+  - dedicated vpc
+  - enterprise ddn
+seoFrontMatterUpdated: false
+---
+
+import Thumbnail from "@site/src/components/Thumbnail";
+
+# How to Provision a Data Plane
+
+## Introduction
+A Data Plane in Hasura provides dedicated infrastructure for running your GraphQL workloads. This guide walks you through the process of provisioning a new Data Plane using Hasura's DDN VPC.
+
+## Prerequisites
+You must have dedicated data plane contacts. If you don't have any data plane contract, reach out to our sales team sales [here](https://hasura.io/contact-us). 
+
+## Provisioning Process
+
+### Step 1: Initiate Data Plane Provisioning
+
+Navigate to the `Data Plane Management` section in your Hasura console. 
+
+<Thumbnail src="/img/data-plane/data-plane-management.png" alt="Data Plane" width="1000px" />
+
+You'll see a list of all your contracted Data Planes. Click the `Provision` button to start the provisioning process for the data plane you want to provision.
+
+<Thumbnail src="/img/data-plane/pending-provision-data-planes.png" alt="Data Plane" width="1000px" />
+
+### step 2: Fill Out the VPC Deployment Form
+
+To start provisioning for a new Data Plane, you'll need to provide the following information in the deployment form:
+
+<Thumbnail src="/img/data-plane/vpc-form.png" alt="Data Plane" width="1000px" />
```

Comment:
Let's change the domain to `private-ddn.hasura.app`

Reply:
Done!